tests/rados/pool_tests.py

```diff
@@ -1131,6 +1143,114 @@ def run(ceph_cluster, **kw):
                 rados_obj.log_cluster_health()
         return 0
 
+    if config.get("Verify_osd_in_out_behaviour"):
+        log.info(
+            "Test to verify the behaviour of the OSD when it is marked out and back in to the cluster"
+        )
+        try:
+            osd_tree = rados_obj.run_ceph_command(cmd="ceph osd tree")
+            osd_list = []
+            for entry in osd_tree["nodes"]:
+                if entry["id"] > 0:
+                    osd_list.append(entry["id"])
```

Comment:
What exactly are we trying to achieve here?

Reply:
A crude way to fetch all the OSDs on the cluster.

Reply:
Use method : def get_active_osd_list(self) here once the PR : https://github.com/red-hat-storage/cephci/pull/3806/ is merged.